docs/user/dql/functions.rst

```diff
@@ -234,9 +234,63 @@ CEIL
 Description
 >>>>>>>>>>>
 
-Specifications:
+Usage: CEIL(a) takes the ceiling of value a.
+
+Limitation: CEIL only works as expected when IEEE 754 double type displays decimal when stored.
+
+Argument type: a: double
```

Comment:
```suggestion
Argument type: <any numeric type>
```
I don't know how to write better - list all types or leave a placeholder. The function can round an integer too, result would be `LONG` anyway.
Please, update for all functions.

Reply:
Addressed in b5fd1208eb4d4012ce8a7fa16b93a2b50ac4eaf1